src/api/helpers/Token.helper.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable require-jsdoc */
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+class Tokenizer {
+  static async generateToken(user) {
+    const { id } = user;
+    const token = await jwt.sign({ userId: id }, process.env.SECRET_KEY, { expiresIn: '2 days' });
```

Comment:
can you include all user information (except the password) in the payload so that if we need for example a user first name and last name we don't have to go in the database?